portal/src/main/webapp/auth_include.jsp

```diff
@@ -10,7 +10,7 @@
     else if (authenticationMethod.equals("googleplus") ||
 	    		authenticationMethod.equals("saml") ||
 	    		authenticationMethod.equals("ad") ||
-	    		authenticationMethod.equals("social_auth")) {
```

Comment:
@alisman looks like this file is not used. should I delete it?

Reply:
yes you can delete